.ci/Jenkinsfile_coverage

```diff
@@ -0,0 +1,47 @@
+#!/bin/groovy
+
+library 'kibana-pipeline-library'
+kibanaLibrary.load()
+
+stage("Kibana Pipeline") { // This stage is just here to help the BlueOcean UI a little bit
+  timeout(time: 180, unit: 'MINUTES') {
+    timestamps {
+      ansiColor('xterm') {
+        catchError {
+          parallel([
```

Comment:
Can you wrap the parallel block with this? The env vars you need should be provided here

```
withEnv([
	'CODE_COVERAGE=1',
	'NODE_ENV=test',
]) {
	parallel(...)
}
```